drivers/adxl345/adxl345.c

```diff
@@ -43,6 +44,12 @@ int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
     /* get device descriptor */
     dev->params= *params;
```

Comment:
this seems wrong, assigment of `adx1345_params_t **` to `adx1345_params_t`? If you do it like this, I think what you want to do is
```c
memcpy(&dev->params, params, sizeof(adx1345_params_t));
```